src/ui/settings/ViewProfile.qml

```diff
@@ -23,4 +23,8 @@ Item {
         source: "qrc:/ui/authenticationInApp/ViewReauthenticationFlow.qml"
         anchors.fill: parent
     }
+
+    Component.onCompleted: {
+        Sample.viewProfileOpened.record();
```

Comment:
This is kind of a dup of `ProfileFlow + loading` right?

Reply:
You’re right that it’s a dup. It should be covered by `ProfileFlow.state == StateReady`.